scala-protoc-plugins/scala-akka/AkkaGrpcServicePrinter.scala

```diff
@@ -12,91 +12,105 @@ final class AkkaGrpcServicePrinter(
 )(implicit descriptorImplicits: DescriptorImplicits) {
   import descriptorImplicits._
 
-  private val streamObserver = "_root_.io.grpc.stub.StreamObserver"
-  private val killSwitchName = s""""${service.getName}KillSwitch ${System.nanoTime()}""""
+  private val StreamObserver = "_root_.io.grpc.stub.StreamObserver"
```

Comment:
For the sake of Future reviews I would really welcome a simple textual test of this code generation, in order to:
* being able for normal humans to review this only by looking at the PR (at least I was unable)
* have textual blueprint in code about what we are going to generate
* alternatively switching to a big """ string interpolation\

Not sure we should do this as part of this PR, but since I have already concerns with the generated code, which might be requiring multiple change/review cycles with possibly multiple people, I would consider it.

Reply:
There are significant changes needed to make such a test work and considering the urgency and ever-increasing scope of this PR, I wouldn't do it here. I have pushed a version addressing your suggestions which makes the code generator very simple. Let me know if it's clearer now